app/src/main/java/com/example/parkingsystem/mvp/view/ActivityView.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.parkingsystem.mvp.view
+
+import android.app.Activity
+import android.content.Context
+import java.lang.ref.WeakReference
+
+open class ActivityView(activity: Activity) {
+    private val activityRef: WeakReference<Activity> = WeakReference(activity)
+
+    val activity: Activity?
+        get() = activityRef.get()
+
+    val context: Context?
+        get() = activity
+
+}
```

Comment:
to avoid this error add an empty line at the end of the file

Reply:
Done 👍